admiral/pkg/controller/util/migration.go

```diff
@@ -45,15 +45,15 @@ func UpdateEndpointsForDeployToRolloutMigration(serviceInstance map[string]*k8sV
 					Address:  rolloutFqdn,
 					Locality: ep.Locality,
 					Ports:    meshPorts[common.Rollout],
-					Labels:   map[string]string{"type": common.Rollout},
+					Labels:   map[string]string{"type": common.Rollout, "security.istio.io/tlsMode": "istio"},
 				}
 				uniqueEndpointsList = append(uniqueEndpointsList, rolloutEp)
 				requiredServices = append(requiredServices, serviceInstance[common.Rollout])
 			}
 		} else {
 			// TODO: check when will this be applicable, and then
 			// update the required service accordingly
-			ep.Labels = nil
+			delete(ep.Labels, "type")
```

Comment:
what this change?

Reply:
Have to delete the type label post migration is done
Earlier labels were being set to nil
Now deleting the type=depl/rollout label specifically